### ISSUE TITLE
Build fails: undefined: unix.Major and unix.Minor

### ISSUE THREAD

**Issue by lukas-vlcek** [openshift/imagebuilder#118]
Trying to build on Mac and running into:

```
$ pwd
/Users/lvlcek/go/src/github.com/openshift

$ go get github.com/openshift/imagebuilder
# github.com/openshift/imagebuilder/vendor/github.com/fsouza/go-dockerclient/internal/archive
imagebuilder/vendor/github.com/fsouza/go-dockerclient/internal/archive/archive_unix.go:71:25: undefined: unix.Major
imagebuilder/vendor/github.com/fsouza/go-dockerclient/internal/archive/archive_unix.go:72:25: undefined: unix.Minor

$ go version
go version go1.11.4 darwin/amd64

$ uname
Darwin
```

**Comment by rhatdan** [comment]
Does not seem like imagebuilder has been ported to non linux platforms.  What do you plan on doing with it?

**Comment by lukas-vlcek** [comment]
@rhatdan You are running into the same issue?

**Comment by nalind** [comment]
> @rhatdan You are running into the same issue?
> When I was investigating this I seemed to me that it might be relevant to versioning of `golang.org/x/sys` library...

This does look like a build error that would be fixed by pulling in a newer `golang.org/x/sys` package than we're using now.

**Comment by lukas-vlcek** [comment]
I do not think switching to `go1.10` helps. I installed `gvm` and `go1.10` via it and here is what I can see now:

```
$ gvm list

gvm gos (installed)
=> go1.10
   system

$ cd $GOPATH && pwd
/Users/lvlcek/.gvm/pkgsets/go1.10/global

$ go get github.com/openshift/imagebuilder
# github.com/openshift/imagebuilder/vendor/github.com/fsouza/go-dockerclient/internal/archive
src/github.com/openshift/imagebuilder/vendor/github.com/fsouza/go-dockerclient/internal/archive/archive_unix.go:71:25: undefined: unix.Major
src/github.com/openshift/imagebuilder/vendor/github.com/fsouza/go-dockerclient/internal/archive/archive_unix.go:72:25: undefined: unix.Minor
```

**Comment by wjs9715** [comment]
Any luck with this? I get the same thing. @lukas-vlcek Are you using Mojave? It worked for me in High Sierra. Now Im in Mojave and installation failing.

**Comment by lukas-vlcek** [comment]
@wjs9715 unfortunately no improvement on my side. I am on Mojave 10.14.2 (18C54). At this moment I am running this code from Fedora 28.

**Comment by lisa** [comment]
I've also run into this and worked around with brute force `dep ensure -update`. However, I don't know how safe that is since it's a wide brush. `make test` returns true, but I'm not familiar with this project well enough to know if that is sufficient.

**Comment by nalind** [comment]
I think the just-merged #122 pulled in the fix for this.

**Comment by lukas-vlcek** [comment]
I can no longer replicate this when using:
```
$ gvm list

gvm gos (installed)

   go1.10
=> go1.11.13
   system

$ ls -la | grep imagebuilder

$ go get -u github.com/openshift/imagebuilder/cmd/imagebuilder

$ ls -la | grep imagebuilder
drwxr-xr-x  27 lvlcek  staff  864 Jan 22 13:25 imagebuilder

$ imagebuilder -version
1.1-dev
```

Seems like it has been fixed.
